pysat/tests/test_custom.py

```diff
@@ -151,11 +148,11 @@ def custom1(inst):
             return out
         self.add(custom1, 'add')
         self.testInst.load(2009, 1)
-        ans = (((self.testInst.data['doubleMLT'] == 2.0 *
-                 self.testInst['mlt']).all()) &
-               ((self.testInst.data['tripleMLT'] == 3.0 *
-                 self.testInst['mlt']).all()))
-        assert ans
+        ans1 = (self.testInst.data['doubleMLT'] == 2.0 *
```

Comment:
This was a poor style I started. the ans1 and ans2 should be removed in favor of direct asserts, ans1 = should be assert instead.

Reply:
updating these to use separate assert statements for each statement